source/guides/stacks/dbrs.rst

```diff
@@ -52,10 +52,10 @@ The release announcement for each release provides more detail about the stack f
 Hardware Requirements
 *********************
 
-* Intel® Xeon Scalable Platform with Intel® C620 chipset series
-* 2nd Gen Intel® Xeon Scalable processor CPU (Intel® Optane™ PMM-enabled stepping) Provides cache & memory control.  Intel® Optane™  PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or Gold processors.
+* Intel Xeon Scalable platform with Intel® C620 series chipset
+* 2nd Gen Intel Xeon Scalable processor CPU (Intel® Optane™ PMem-enabled stepping). Provides cache & memory control.  Intel Optane PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or Intel® Xeon® Gold processors.
```

Comment:
```suggestion
* 2nd Gen Intel Xeon Scalable processor CPU (Intel® Optane™ PMem-enabled stepping). Provides cache & memory control.  Intel Optane PMem works only on systems powered by 2nd Generation Intel® Xeon® Platinum or the Intel® Xeon® Gold processor.
```

Reply:
I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).